server/src/instant/model/app.clj

```diff
@@ -36,6 +36,38 @@
 (defn get-by-id! [params]
   (ex/assert-record! (get-by-id params) :app {:args [params]}))
 
+(defn list-account-apps
+  ([user-id] (list-account-apps aurora/conn-pool user-id))
+  ([conn user-id]
```

Comment:
nit: this would only list the apps where the user is the creator. afaik this is good, but would just call it `list-created-apps` with `creator-id`